src/main/java/com/assignment/Assignment2.java

```diff
@@ -0,0 +1,11 @@
+package com.assignment;
+
+public class Assignment2 {
+
+    public String returnReplacedSubString(String originalString,String subStringToBeReplaced, String replacement){
```

Comment:
The naming convention for a method should be :
get<method-name> -> if its returning something
set<method-name> -> if its setting something